src/dump.ts

```diff
@@ -3,8 +3,10 @@
 */
 import * as request from "request-promise-native"
 import * as vscode from "vscode"
+import { parseStringPromise } from "xml2js"
```

Comment:
Alphabetical order

Reply:
Would it not be better separating the whole export with the specific named exports? (See extension.ts for example)

Reply:
I forgot that JavaScript imports are always ugly, been working in a lot of other stuff recently. Never mind this.